### ISSUE TITLE
How do you solve the noise/artefact problem ?

### ISSUE THREAD

**Issue by JunbinWang** [nagadomi/waifu2x#300]
Hi,

I am very interested about how you train the model to solve the image artifact problem caused by image compression (jpeg images) ?

I try some other SR models before, most of them can only handle png images and performs very bad in jpeg format image.

Best Regard,
Even.

**Comment by nagadomi** [comment]
simply train jpeg denoising.

denoise:
input image -> add jpeg noise -> CNN -> output image
loss = MSE(input image, output image)
SR+denoise:
input image -> downsampling -> add jpeg noise -> CNN(upsampling) -> output image
loss = MSE(input image, output image)
